### PR TITLE
Refactor/load background

### PR DESCRIPTION
**Pull request by aringocode**
The first time this file is loaded, it should have the correct background.

I know I will need to update this in all the other repos.

### CODE REVIEW

**Comment by aringocode** [comment]
Network: Regular 3G

How it looks like with brighter colours. Red is static `backgound-color: pink;`, blue is used via variables `background-color: $link-color;`
dcol



How it looks with normal colors
stcol

File: public/styles.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+body {
+    background-color: pink;
```
Comment (Pe5h4) [inline public/styles.css]: @Aringoaway We dont want to have a pink color as a background

File: src/index.js [hunk 2]
```diff
@@ -34,6 +34,10 @@ const ConfigDefaults = {
 
 const modules = [];
 
+window.addEventListener('load', (event) => {
```
Comment (Pe5h4) [inline src/index.js:38]: @Aringoaway Write a description here pls

File: public/index.html [hunk 3]
```diff
@@ -24,6 +24,7 @@
 		<link rel="icon" type="image/png" sizes="96x96" href="./media/teskalabs/img/favicon/favicon-96x96.png">
 		<link rel="icon" type="image/png" sizes="16x16" href="./media/teskalabs/img/favicon/favicon-16x16.png">
 		<link rel="manifest" href="./media/teskalabs/img/favicon/manifest.json">
+		<link rel="stylesheet" href="styles.css">
```
Comment (Pe5h4) [inline public/index.html:27]: @Aringoaway Where is the spinner?

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
@Aringoaway Pls add changes to changelog

**Comment by aringocode** [comment]
@Pe5h4  Add spinner 
Network: Regular 2G
spinner

**Review by Pe5h4 — APPROVED** [review]
@Aringoaway Ok, this looks better

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,10 @@
 
 ## Release Candidate
 
+## Refactoring
```
Comment (Pe5h4) [inline CHANGELOG.md:5]: @Aringoaway You have `Refactoring` two times in the CHANGELOG, pls redo

**Review by Pe5h4 — CHANGES_REQUESTED** [review]

**Review by ateska — APPROVED** [review]

**Review by Pe5h4 — APPROVED** [review]
@Aringoaway Pls resolve conflicts before merging

**Pull request merged**